velox/exec/MemoryReclaimer.cpp

```diff
@@ -66,6 +66,106 @@ void MemoryReclaimer::abort(
   });
 }
 
+/*static*/ std::unique_ptr<memory::MemoryReclaimer> NodeMemoryReclaimer::create(
```

Comment:
/* static */ is the new convention to follow?

Reply:
This is existing code style?

Reply:
I thought we used to do // static on the top. If this is the standard then please ignore this comment.